ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,264 @@
+import { left } from "fp-ts/lib/Either";
+import { Option, some } from "fp-ts/lib/Option";
+import I18n from "i18n-js";
+import * as pot from "italia-ts-commons/lib/pot";
+import { Tab } from "native-base";
+import * as React from "react";
+import { Animated, Platform, StyleSheet, TouchableOpacity } from "react-native";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { ServicePublic } from "../../../definitions/backend/ServicePublic";
+import IconFont from "../../components/ui/IconFont";
+import { userMetadataUpsert } from "../../store/actions/userMetadata";
+import { Organization } from "../../store/reducers/entities/organizations/organizationsAll";
+import {
+  localServicesSectionsSelector,
+  organizationsOfInterestSelector,
+  ServicesSectionState
+} from "../../store/reducers/entities/services";
+import { readServicesByIdSelector } from "../../store/reducers/entities/services/readStateByServiceId";
+import { profileSelector } from "../../store/reducers/profile";
+import { GlobalState } from "../../store/reducers/types";
+import {
+  UserMetadata,
+  userMetadataSelector
+} from "../../store/reducers/userMetadata";
+import { makeFontStyleObject } from "../../theme/fonts";
+import customVariables from "../../theme/variables";
+import { getLogoForOrganization } from "../../utils/organizations";
+import { isTextIncludedCaseInsensitive } from "../../utils/strings";
+import ChooserListContainer from "../ChooserListContainer";
+import { withLightModalContext } from "../helpers/withLightModalContext";
+import { LightModalContextInterface } from "../ui/LightModal";
+import OrganizationLogo from "./OrganizationLogo";
+import ServicesSectionsList from "./ServicesSectionsList";
+
+type OwnProps = Readonly<{
+  heading: string;
+  isLocal?: boolean;
+  sections: ReadonlyArray<ServicesSectionState>;
+  isRefreshing: boolean;
+  onRefresh: (hideToast: boolean) => void;
+  onServiceSelect: (service: ServicePublic) => void;
+  handleOnLongPressItem: () => void;
+  isLongPressEnabled: boolean;
+  dispatchUpdateOrganizationsOfInterestMetadata?: (
```

Comment:
```suggestion
  onUpdateOrganizationsOfInterestMetadata?: (
```
Maybe using _dispatch_ could be confusing with actions dispatching

Reply:
yes, updated within https://github.com/teamdigitale/io-app/pull/1409/commits/97303adcf1dbbde16b4c8a6be32732937f18f978